README.md

```diff
@@ -2,177 +2,27 @@
 
 ![CI](https://github.com/tribe29/grafana-checkmk-datasource/actions/workflows/node.js.yml/badge.svg)
 
-This data-source plugin is a complete rewrite from the previous connector. It has
-undergone a new architectural design and is not backwards compatible to the
-previous version. We nevertheless provide an upgrade procedure to assist you on
-the transition.
+This is a [Grafana][1] [data source][2] plugin for working with checkmk metrics
+in Grafana.
 
-This project has entered the Beta testing phase.
+The plugin requires checkmk >= 2.1.0 and Grafana >= 8.0  
+(It is possible query data from checkmk > 2.0.0p20 but it's not recommended.)
 
-## Requirements
+Both Checkmk Enterprise Edition and Checkmk Raw Edition are supported by this
+plugin, but with different query interface as the Enterprise Edition has a
+better filtering engine.
 
-- Checkmk >= 2.0.0p20
-- Grafana >= 8.0
+Please consult chapter ["Integrating Checkmk in Grafana"][3] in the official
```

Comment:
-> Please consult the chapter [Integrating Checkmk in Grafana][3] in the official